src/components/Modal/FeedbackModal/index.tsx

```diff
@@ -0,0 +1,156 @@
+import Modal from "..";
+import * as S from "./styled";
+import Close from "@assets/icons/close.svg";
+import SosoOn from "@assets/icons/soso-on.svg";
+import SosoOff from "@assets/icons/soso-off.svg";
+import BadOn from "@assets/icons/bad-on.svg";
+import BadOff from "@assets/icons/bad-off.svg";
+import GoodOn from "@assets/icons/good-on.svg";
+import GoodOff from "@assets/icons/good-off.svg";
+import { useState } from "react";
+import { instance } from "../../../apis/axios";
+
+interface FeedbackModalProps {
+  isOpen: boolean;
+  closeModal: () => void;
+}
+
+type ToggleState = {
+  good: boolean;
+  soso: boolean;
+  bad: boolean;
+};
+
+type IconType = keyof ToggleState;
+
+type FeedbackType = "BAD" | "AVERAGE" | "EXCELLENT";
+
+interface FeedbackRequest {
+  type: FeedbackType;
+  message: string;
+}
+
+const MAX_LENGTH = 1000;
+
+const FeedbackModal: React.FC<FeedbackModalProps> = ({
+  isOpen,
+  closeModal,
+}) => {
+  const [toggleStates, setToggleStates] = useState<ToggleState>({
+    good: false,
+    soso: false,
+    bad: false,
+  });
+
+  const handleToggle = (iconType: IconType): void => {
+    setToggleStates((prev) => {
+      // 이미 선택된 아이콘을 다시 클릭한 경우
+      if (prev[iconType]) {
+        return {
+          ...prev,
+          [iconType]: false,
+        };
+      }
+
+      // 다른 아이콘을 클릭한 경우
+      return {
+        good: false,
+        soso: false,
+        bad: false,
+        [iconType]: true, // 새로 선택된 아이콘만 on
+      };
```

Comment:
```suggestion
const defaultFeedbackState = { good: false, soso: false, bad: false };
      return {
        ...defaultFeedbackState,
        [iconType]: true, // 새로 선택된 아이콘만 on
      };
```
`useState 초기값`, `handleToggle 함수 내부`, `handleCloseModal 함수 내부`에서 모두 false인 기본값 상태를 많이 사용하는 것 같아서 `defaultFeedbackState`라는 변수를 만들어서 사용해도 좋을 것 같아요~!

Reply:
오우 이런식으론 생각못했는데 감사해요 !!